integration/containerd/cri/integration-tests.sh

```diff
@@ -44,7 +47,6 @@ CI=${CI:-""}
 
 containerd_shim_path="$(command -v containerd-shim)"
 readonly cri_containerd_repo="github.com/containerd/containerd"
```

Comment:
Since this patch brings `.ci/lib.sh` in, this open-coded string could be converted to
 `$(get_version "externals.containerd.url")` like `install_cri_containerd.sh`
 already does.

Appart from that, this is a very nice simplification. Thanks !

Reply:
Hi @gkurz 

Thanks for your comments, updated.